ledger-core/virtual/integration/convlog/default_others.go

```diff
@@ -7,4 +7,4 @@
 
 package convlog
 
-const UseTextConvLog = false
+const UseTextConvLog = true
```

Comment:
please don't change this constant, you have to use build flags!

Reply:
sure

Reply:
fixed